### PR TITLE
179/fix/missing geo locations

### PR DESCRIPTION
**Pull request by DanielKraft**
This pull request fixes the error that the lat lon enhancer terminates when a 'NoneType' object is accessed.

For testing:
```shell
$ cd <git-repo>/etl
$ /usr/bin/time -v python main.py
```

If you already have the data_extraction data, you can speed up the process by commenting out line 16 in main.py.

Please merge after #206

### CODE REVIEW

**Comment by jonas-hda** [comment]
for the future, please keep your branches 'clean'. branches should always be created from staging and should only contain changes related to your issue

File: etl/data_enhancement/domain/default.py [hunk 1]
```diff
@@ -75,6 +76,5 @@ def rank_tags(file):
 # Enhance the geo_location
 def add_lat_lon(file):
     enhancer = LatLonEnhancer()
-
     for post in file:
-        post = enhancer.enhance(post)
+        enhancer.enhance(post)
```
Comment (FiedlerJan) [inline etl/data_enhancement/domain/default.py:80]: Please remove the return value from enhance. It is not needed anymore.

**Review by FiedlerJan — CHANGES_REQUESTED** [review]

File: etl/data_enhancement/domain/default.py (hunk 1 above)
Comment (DanielKraft) [reply etl/data_enhancement/domain/default.py:80]: Done. You have already changed it.

Comment (FiedlerJan) [reply etl/data_enhancement/domain/default.py:80]: It is not changed. lat_lon_enhancer.py returns post in the enhance function (line 34) which is never used.

Comment (DanielKraft) [reply etl/data_enhancement/domain/default.py:80]: Sorry. I thought I should change something in the displayed line.

**Review by jonas-hda — APPROVED** [review]

**Review by FiedlerJan — APPROVED** [review]

**Pull request merged**